src/features/query-bar/constants.js

```diff
@@ -1,17 +1,22 @@
 // @flow
 
+export const AND: 'AND' = 'AND';
+export const OR: 'OR' = 'OR';
 export const ATTRIBUTE: 'attribute' = 'attribute';
 export const COLUMN_DISPLAY_TEXT: 'columnDisplayText' = 'columnDisplayText';
 export const COLUMN_KEY: 'columnKey' = 'columnKey';
+export const CONNECTOR_NAMES = {
```

Comment:
I'm not sure about the variable name.

It's odd that index 0 means AND, index 1 means OR.

I don't see a purpose for this. What do we lose by removing it?

Reply:
Removing it